core/src/main/scala/kafka/coordinator/GroupCoordinator.scala

```diff
@@ -323,8 +326,11 @@ class GroupCoordinator(val brokerId: Int,
       responseCallback(Errors.GROUP_COORDINATOR_NOT_AVAILABLE.code)
     } else if (!isCoordinatorForGroup(groupId)) {
       responseCallback(Errors.NOT_COORDINATOR_FOR_GROUP.code)
+    } else if (isCoordinatorLoadingInProgress(groupId)) {
+      // the group is still loading, so respond just blindly
+      responseCallback(Errors.NONE.code)
```

Comment:
Yeah, I agree, but I can't actually think of any case where the client wouldn't just ignore it. Guess clients might want to add logging in this case?